COMPILING.md

```diff
@@ -114,6 +114,25 @@ files.
    complete set of built binaries can be found in `./build/bin` once the build
    is complete.
 
+## Compilation on Windows using Visual Studio
+
+Follow these instructions to work on Windows with Visual Studio:
+
+1. First, clone the `cbmc` repository locally.
+2. Open the `cbmc` folder into Visual Studio. Visual Studio 2017 and
```

Comment:
English isn't my first language, so take this with a grain of salt: I think it should be "in," not "into." Perhaps someone whose first language _is_ English can do a proof read?

Reply:
Yes, "into" -> "in"